### PR TITLE
Cns 423 fix naming of project txs

### PR DESCRIPTION
**Pull request by Yaroms**

### CODE REVIEW

File: x/projects/client/cli/tx_add_keys.go [hunk 1]
```diff
@@ -15,18 +15,18 @@ import (
...
 
-func CmdAddProjectKeys() *cobra.Command {
+func CmdAddKeys() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "add-project-keys [project-id] [optional: project-keys-file-path]",
+		Use:   "add-keys [project-id] [optional: project-keys-file-path]",
 		Short: "Add developer/admin keys to an existing project",
-		Long: `The add-project-keys command allows the project admin to add new project keys (admin/developer) to the project.
+		Long: `The add-keys command allows the project admin to add new project keys (admin/developer) to the project.
 		To add the keys you can optionally provide a YAML file of the new project keys (see example in cookbook/project/example_project_keys.yml).
```
Comment (oren-lava) [inline x/projects/client/cli/tx_add_keys.go:23]: The path to the example_project_keys.yml file is old. The right path is cookbook/projects/example_project_keys.yml (note the s in projects)

File: x/projects/client/cli/tx_add_keys.go [hunk 2]
```diff
@@ -15,18 +15,18 @@ import (
...
 		Note that in project keys, to define the key type, you should follow the enum described in the top of example_project_keys.yml.
 		Another way to add keys is with the --admin and --developer flags. Note that the developer must come with its VRF key (see example)`,
 		Example: `required flags: --from <admin-key> (the project's subscription address is also considered admin)
 				  
-		lavad tx subscription add-project-keys [project-id] [project-keys-file-path] --from <admin-key>
-		lavad tx subscription add-project-keys [project-id] --admin <other-admin-key> --admin <another-admin-key> --developer <developer-key>:<developer-vrfpk> --from <admin-key>`,
+		lavad tx project add-keys [project-id] [project-keys-file-path] --from <admin-key>
+		lavad tx project add-keys [project-id] --admin <other-admin-key> --admin <another-admin-key> --developer <developer-key>:<developer-vrfpk> --from <admin-key>`,
```
Comment (oren-lava) [inline x/projects/client/cli/tx_add_keys.go:29]: in the usage of the developer key flag, there is a use of vrfpk. Is it still relevant? if not, the comment and the code that handles the developer flag should be changed

File: x/projects/client/cli/tx_set_admin_policy.go [hunk 3]
```diff
@@ -13,13 +13,13 @@ import (
...
 var _ = strconv.Itoa(0)
 
-func CmdSetAdminPolicy() *cobra.Command {
+func CmdSetPolicy() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "set-admin-policy [project-index] [policy-file-path]",
+		Use:   "set-policy [project-index] [policy-file-path]",
 		Short: "set policy to a project",
-		Long:  `The set-admin-policy command allows a project admin to set a new policy to its project. The policy file is a YAML file (see cookbook/project-policies/example.yml for reference). The new policy will be applied from the next epoch.`,
+		Long:  `The set-policy command allows a project admin to set a new policy to its project. The policy file is a YAML file (see cookbook/project-policies/example.yml for reference). The new policy will be applied from the next epoch.`,
```
Comment (oren-lava) [inline x/projects/client/cli/tx_set_admin_policy.go:20]: The path to the example policy file is old. The right path is cookbook/projects/example_policy.yml

File: x/projects/client/cli/tx_set_subscription_policy.go [hunk 4]
```diff
@@ -17,7 +17,7 @@ var _ = strconv.Itoa(0)
 func CmdSetSubscriptionPolicy() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "set-subscription-policy [project-index] [policy-file-path]",
-		Short: "set policy to a project",
+		Short: "set subscription policy to a project",
 		Long:  `The set-subscription-policy command allows the project's subscription consumer to set a new policy to its subscription which will affect some/all of the subscription's projects. The policy file is a YAML file (see cookbook/project-policies/example.yml for reference). The new policy will be applied from the next epoch.`,
```
Comment (oren-lava) [inline x/projects/client/cli/tx_set_subscription_policy.go:21]: same comment about old example policy file path

**Review by oren-lava — CHANGES_REQUESTED** [review]

File: x/projects/client/cli/tx_add_keys.go (hunk 2 above)
Comment (Yaroms) [reply x/projects/client/cli/tx_add_keys.go:29]: vrf removal is on another task :)

**Pull request merged**